src/cluster_legacy.c

```diff
@@ -1011,10 +1011,10 @@ void clusterInit(void) {
      * us from trying to use a too-high cluster port number. */
     int port = defaultClientPort();
     if (!server.cluster_port && port > (65535-CLUSTER_PORT_INCR)) {
-        serverLog(LL_WARNING, "Redis port number too high. "
+        serverLog(LL_WARNING, "Valkey port number too high. "
```

Comment:
I am struggling with this: Valkey or Server @valkey-io/core-team

Reply:
Suggest we use "Server port number" instead.

Reply:
The question is whether we need redis compatibility in log entries. We've discussed adding a compatibility switch (compile time or runtime config). No decision yet though.